content/docs/examples/multicluster/split-horizon-eds/index.md

```diff
@@ -17,22 +17,21 @@ By following the instructions in this example, you will setup a two cluster mesh
   link="./diagram.svg"
   caption="Single Istio control plane topology spanning multiple Kubernetes clusters with Split-horizon EDS configured" >}}
 
-The `local` cluster will run Istio Pilot and the other Istio control plane components while the `remote` cluster only runs Istio Citadel,
+The `cluster1` cluster will run the full Istio control plane components while `cluster2` only runs Istio Citadel,
```

Comment:
How about:

The primary cluster, `cluster1`, runs the full set of Istio control plane components ...